jwst/resample/resample.py

```diff
@@ -65,9 +65,14 @@ def __init__(self, input_models, output=None, single=False, blendheaders=True,
         self.pixfrac = pixfrac
         self.kernel = kernel
         self.fillval = fillval
-        self.weight_type = weight_type
+        self.weight_type = wht_type
         self.good_bits = good_bits
 
+        log.info(f"Driz parameter kernal: {self.kernel}")
```

Comment:
typo: "kernal" should be "kernel"

Also, are these the values coming from the spec, i.e. before the drizpars ref file has been loaded?

Reply:
I added printing the driz parameters because they can be filled from either the reference file or the driz ref files.
These values are the final ones that are used in resample. So they are after the drizpars ref file has been loaded.